src/main/java/com/google/api/generator/engine/writer/JavaWriterVisitor.java

```diff
@@ -62,14 +62,15 @@
 
   private static final String COLON = ":";
   private static final String COMMA = ",";
-  private static final String BLOCK_COMMENT_START = "/**";
```

Comment:
Please separate the changes in #168 from this PR.

Reply:
Hold this PR until #168 is merged.